scripts/byron-to-alonzo/simple-tx.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+# This script creates, signs, and submits a simple transaction in the Alonzo era.
+# We simply generate a new user and send him some money from a pre-build utxo account.
+# To use: start an alonzo cluster and then run as ./scripts/byron-to-alonzo/simple-tx.sh
+
+ROOT=example
+TESTNET=42
+WORK=simple-tx-files
+CLI=cardano-cli
+pushd ${ROOT}
+mkdir ${WORK}
+sleep 15 # Wait for things to initialize and the socket to be ready
```

Comment:
Is the intention here that you want to run the mkfiles.sh and this script at the same time? Rather than manually waiting for the cluster to start?

Reply:
Yes, I'll omit that and just put a note at the top to run the script after the cluster has started working and you can query the tip from `node-bft1`, say.